.github/workflows/installer-windows-parula.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     - run: cd e2; npm install --legacy-peer-deps
     - run: cd e2; npm run build:release:win
       env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        GITHUB_TOKEN: ${{ secrets.PARULA_GH_TOKEN }}
```

Comment:
I am not sure how that would work. The `GITHUB_TOKEN` is a special variable that GitHub sets. I wouldn't know where to get it from. If we can generate one, then I would know how to set it in the other repo. There's a page in the settings to set things like `PARULA_GH_TOKEN`.